pkg/test/constants/presubmitconstants.go

```diff
@@ -32,6 +32,7 @@ var (
 		"accesscontextmanager": {"accesscontextmanagerserviceperimeter"},
 		"alloydb":              {"fullalloydbcluster"},
 		"apigee":               {"apigeeenvironment"},
+		"apikeys":              {"apikeyskeybasic"},
```

Comment:
I think we may not want to add v1alpha1 resources into presubmit (and get distracted by its failures)?

Reply:
So this is needed because of TestAllServicesInMap.

I do think we want to test it because we're actively developing it, and it's an important evaluation of a new approach.  The API itself is relatively old I believe, so I don't think we'll see too many service flakes.

Reply:
Sure, I'm okay with us keeping it as is. Though we may want to re-evaluate it if it adds more maintenance burden than expected.